components/navigation/ui/src/main/resources/PhenoTips/DBWebHomeSheet.xml

```diff
@@ -420,6 +420,10 @@ document.observe('xwiki:livetable:newrow', function(tableEvent){
   display: none;
 }
 
+xwiki-livetable td.actions {
```

Comment:
Missing a dot at the start of the line.

Reply:
Ouch, wrong copy/paste... Fixed, thanks!